SPECS/qemu-dp.spec

```diff
@@ -98,10 +98,16 @@ Patch87: make-blockdev-snapshot-use-same-node-name.patch
 Patch88: use_existing_io_context.patch
 Patch89: use_libaio_by_default.patch
 Patch90: Use_the_legacy_grant_copy_ioctl
+
+#XCP-ng patches
```

Comment:
I just updated the conventions with an example. I added a space after the # (prettier than what I first wrote) and removed the empty line after the comment:
```
# XCP-ng patches
Patch1000: qemu-dp-2.10.2-add-rbd-support.XCP-ng.patch

BuildRequires:...
```